.github/ISSUE_TEMPLATE/new-patch-release.md

```diff
@@ -19,8 +19,12 @@ Please do not remove items from the checklist
   - [ ] Upload release artefacts generated by `prepare-release.sh` script above to the draft release
 - [ ] An OWNER runs
      `git tag -s -m "NFD release $VERSION" $VERSION`
+- [ ] An OWNER runs
+     `cd api/nfd`
```

Comment:
Should be able to create the tag in the repo root

```suggestion
- [ ] An OWNER runs
     `cd api/nfd`
```

Reply:
Done